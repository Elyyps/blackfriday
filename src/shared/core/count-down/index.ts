export * from "./components/count-down.component";
