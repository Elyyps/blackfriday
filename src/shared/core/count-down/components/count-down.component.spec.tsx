import React from "react";
import { shallow } from "enzyme";
import { CountDownComponent } from "./count-down.component";

describe("[CountDown]", () => {
  it("should render component without crashing", () => {
    const renderedComponent = shallow(<CountDownComponent />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
