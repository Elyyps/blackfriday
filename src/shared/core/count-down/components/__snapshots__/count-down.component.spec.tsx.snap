// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[CountDown] should render component without crashing 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
